src/panel/direction/DirectionInput.jsx

```diff
@@ -70,12 +71,12 @@ class DirectionInput extends React.Component {
       }
 
       if (selectedPoi.status === navigatorGeolocationStatus.FOUND) {
-        this.props.onChangePoint(selectedPoi.getInputValue(), selectedPoi);
+        this.props.onChangePoint(await getInputValue(selectedPoi), selectedPoi);
```

Comment:
The bug on direction inputs was here, and the same below: I forgot the `await`, so the field displayed `[object Promise]` instead of the POI label.